src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedRecipeDurationTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.recipe.testutil.Assert.assertThrows;
+import static seedu.recipe.testutil.TypicalRecipes.CACIO_E_PEPE;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.recipe.RecipeDuration;
+import seedu.recipe.model.recipe.unit.TimeUnit;
+
+public class JsonAdaptedRecipeDurationTest {
+    private static final RecipeDuration RECIPE_DURATION = new RecipeDuration(30, new TimeUnit("minutes"));
+    private static final double TIME = 30;
+    private static final TimeUnit TIME_UNIT = new TimeUnit("minutes");
+    private static final JsonAdaptedTimeUnit JSON_ADAPTED_TIME_UNIT = new JsonAdaptedTimeUnit(TIME_UNIT);
+
```

Comment:
Accept plural or non plural for `1` here? @filbertphang @alson001 

i.e. `1 minute` or `1 minutes`

Reply:
Realised no way for it throw IllegalValueException as the check occurs earlier. This plural or non plural should be done in model test.